dask/dataframe/io/tests/test_parquet.py

```diff
@@ -613,3 +613,14 @@ def test_writing_parquet_with_compression(tmpdir, compression, engine):
     ddf.to_parquet(fn, compression=compression, engine=engine)
     out = dd.read_parquet(fn, engine=engine)
     assert_eq(out, df, check_index=(engine != 'fastparquet'))
+
+
+def test_writing_parquet_with_unknown_kwargs(tmpdir, engine):
+    fn = str(tmpdir)
+
+    df = pd.DataFrame({'x': ['a', 'b', 'c'] * 10,
+                       'y': [1, 2, 3] * 10})
+    ddf = dd.from_pandas(df, npartitions=3)
+
+    with pytest.raises(TypeError):
+        ddf.to_parquet(fn,  engine=engine, unknown_key='unknown_value')
```

Comment:
This only tests passing in an unexpected keyword argument, which is a different issue than the one you experienced, with passing in backend-specific keyword arguments. Could you add a test with something specific to each backend (e.g. `use_dictionary` for pyarrow)?